craft_cli/messages.py

```diff
@@ -39,23 +41,51 @@ class _MessageInfo:
 # the different modes the Emitter can be set
 EmitterMode = enum.Enum("EmitterMode", "QUIET NORMAL VERBOSE TRACE")
 
+# the limit to how many log files to have
+LOG_FILES_LIMIT = 5
+
```

Comment:
This renders nicely in sphinx

```suggestion
LOG_FILES_LIMIT = 5
"""The limit to how many log files to have."""
```

I might personally prefer MAX over LIMIT

Reply:
Agree with Sergio, `MAX_LOG_FILES` could be better here.

Reply:
Changed to `_MAX_LOG_FILES`, to also make it private (not intended to be part of the public API)